### DISCUSSION TITLE
如何让mDNS交给系统处理

### DISCUSSION THREAD

**Discussion by posthumz** [MetaCubeX/mihomo#1407] [Q&A]
不开启代理时系统可以正确解析`设备名.local`
开启代理后`设备名.local`由clash meta解析不成功，提示`设备名.local:3000 error: dns resolve failed: couldn't find ip`
尝试用`fake-ip-filter: [+.local]`和`nameserver-policy: { geosite:private: system }`都不能正确让系统解析

补充：Windows环境下

**Comment by only52607** [comment]
找到解决方法了吗

**Reply by posthumz** [reply]
目前在使用`系统代理绕过设置`里面加了个`*.local`可以暂时解决问题（tun模式下似乎没有这个问题）

不过还是希望有个更好的解决方法

**Reply by only52607** [reply]
方法可行，感谢

**Reply by only52607** [reply]
> 目前在使用`系统代理绕过设置`里面加了个`*.local`可以暂时解决问题（tun模式下似乎没有这个问题）
> 
> 不过还是希望有个更好的解决方法

tun模式下使用mDNS还有什么额外的配置吗。代理模式下可以添加绕过配置解决，然而我开启tun模式之后mDNS却失效了。

**Reply by posthumz** [reply]
> tun模式下使用mDNS还有什么额外的配置吗。代理模式下可以添加绕过配置解决，然而我开启tun模式之后mDNS却失效了。

没有额外，可能和你的DNS配置或者规则有关

在Powershell里面`Resolve-DnsName 设备名.local`看看是不是DNS问题

不是的话就是你的规则了，我规则里有一条`GEOSITE,private,直连`

**Reply by only52607** [reply]
解决了，添加`fake-ip-filter: *.local`即可
